### ISSUE TITLE
Feature Request: Alarm Mode

### ISSUE THREAD

**Issue by RobertCordingly** [lukash/refloat#25]
I see in Floaty you have the option to lock your board and honk the horn. If those two features can be combined together to create an alarm mode that would be awesome!

Basically you would lock your board and then if the board is moved a significant amount the board would repeatedly honk the horn at maximum volume

**Comment by lukash** [comment]
Yeah, this feature is on my list. I wanted to do it more so on controllers that have a non-latching switch and disable turning the board off, because otherwise you can just turn the board off (the thief would still need to know / realize, but yeah). I'm too busy implementing more important things now, no idea when I'll be able to get to this.
